supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -827,16 +832,28 @@ class PipelinesApiAsyncBackendJobExecutionActor(override val standardParams: Sta
         throw new RuntimeException(s"handleExecutionSuccess not called with RunStatus.Success. Instead got $unknown")
     }
 
-  override def getStartAndEndTimes(runStatus: StandardAsyncRunState): Option[StartAndEndTimes] =
-    runStatus match {
-      case terminalRunStatus: TerminalRunStatus if terminalRunStatus.eventList.nonEmpty =>
```

Comment:
I don't believe bard will see any changes here, even though this function may return slightly different things than it used to depending on when it is called. 

Because we only call `getStartAndEndTimes` (and do the whole bard thing) on the transition into a terminal status, all of the same information will be present at the time this is called, and therefore bard will see the same start and end times.  

If bard relied on a polling approach like cost does, then this change would certainly affect bard.